src/Login/action_types.js

```diff
@@ -0,0 +1,6 @@
+export default {
+    USERNAME: 'LOGIN_USERNAME',
```

Comment:
The reason to do it this way is so that you can import all login actions together. I find importing individual action types to be a little cumbersome. It requires the imports to be changed every time you add a new action.

There's another method may be better.

```
export const USERNAME = 'LOGIN_USERNAME';
---
import * as LOGIN from './action_types';
```

I think this will still allow you to reference the action as LOGIN.USERNAME.